assets/js/modules/tagmanager/setup.js

```diff
@@ -406,12 +428,52 @@ class TagmanagerSetup extends Component {
 		);
 	}
 
-	render() {
+	/**
+	 * Render Error or Notice format depending on the errorCode.
+	 */
+	renderErrorOrNotice() {
 		const {
-			error,
-			message
+			errorCode,
+			errorMsg,
 		} = this.state;
 
+		const {
+			onSettingsPage,
+		} = this.props;
+
+		if ( ! errorCode ) {
+			return null;
+		}
```

Comment:
We should check for a non-zero length error message here as well rather than in both conditions below.